src/module/actor/base.ts

```diff
@@ -1328,43 +1328,49 @@ class ActorPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e | n
             persistentDamage.length > 0 ? await this.createEmbeddedDocuments("Item", persistentDamage) : []
         ) as ConditionPF2e<this>[];
 
+        const canUndoDamage = hpDamage > 0 || shieldDamage || persistentCreated.length;
```

Comment:
Would this still allow for undoing healing?

Reply:
Oh true, that wouldn't work